src/juju/jimm/index.ts

```diff
@@ -0,0 +1,10 @@
+import JIMMV3 from "./JIMMV3";
+
+export * as JIMMV3 from "./JIMMV3";
```

Comment:
I am not sure if I understood the intention here correctly, but if we want to export the imported `JIMMV3`, shouldn't we have `export { default as JIMMV3 } from "./JIMMV3";` or `export { JIMMV3 };` here instead?

Reply:
I think normally that would be a better idea (or probably forgo the re-export entirely), but I'm inclined to keep it like this so that it's consistent with the other facades from jujulib, so they can all be consumed in the same way e.g. https://github.com/juju/js-libjuju/blob/master/api/facades/client/index.ts.

For the JIMM v3 facade there's only a single export that gets captured by this wild card export, but for JIMM v4 there are more: https://github.com/canonical/juju-dashboard/blob/63230bd08bc31c61ef74fa67ad10dccd7ddb2c16/src/juju/jimm/JIMMV4.ts.